Sources/Sentry/include/module.modulemap

```diff
@@ -0,0 +1,4 @@
+module Sentry {
```

Comment:
This file is necessary for Swift Package Manager tests via `swift test` (for some reason, the umbrella header SPM generates doesn’t include everything), but isn’t required for Xcode to use and build an app with this package.